docs/metasploit-framework.wiki/Rolling-back-merges.md

```diff
@@ -33,7 +33,7 @@ Like so:
 
 ```
 $ git log bad-merge...bad-merge~ --oneline
-3996557 Fix conflcit lib/msf/util/exe.rb
+3996557 Fix conflict lib/msf/util/exe.rb
```

Comment:
This is an actual commit and the typo is present in the merge message. Updating it here introduces an inconsistency with what should be the source of truth.